main.go

```diff
@@ -43,13 +49,49 @@ func onReady(s *discordgo.Session, event *discordgo.Ready) {
 	log.Printf("Logged in as %s\n", event.User.String())
 }
 
+func addGriefer(s *discordgo.Session, m *discordgo.MessageCreate) {
```

Comment:
We're duplicating a bit of logic here for no reason so it'd be better to consolidate this event handler to a single function (the `messageCreate` we already have), and then split the logic out into sub-functions (`handleGriefer(...)`, `handleHello(...)`, etc).